.circleci/test-deploy.yml

```
@@ -26,7 +26,7 @@ release-filters: &release-filters
 jobs:
   test_executor:
     parameters:
-      executor: {type: executor}
+      executor: { type: executor }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)